WcaOnRails/app/models/team.rb

```diff
@@ -17,4 +17,8 @@ def membership_periods_cannot_overlap_for_single_user
       end
     end
   end
+
+  def current_members
```

Comment:
Actually, if we add the `def self.where_current` to TeamMember that I suggested below, I think this could even get cleaned up to

```ruby
has_many :current_members -> { where_current }, class_name: "TeamMember"
```